graphene_django/rest_framework/tests/test_mutation.py

```diff
@@ -268,6 +273,34 @@ class Meta:
     assert result.days_since_last_edit == 4
 
 
+def test_perform_mutate_success_with_enum_choice_field():
+    class ListViewChoiceFieldSerializer(serializers.ModelSerializer):
+        choice_type = serializers.ChoiceField(
+            choices=MyFakeModelWithChoiceField.ChoiceType, required=False
```

Comment:
Based on the issue reported https://github.com/graphql-python/graphene-django/issues/1385 and the DRF docs herehttps://www.django-rest-framework.org/api-guide/fields/#choicefield, shouldn't this be `choices=MyFakeModelWithChoiceField.ChoiceType.choices` instead? (fwiw, the tests passed for me either way when I ran it)

Reply:
@sjdemartini apologies, its been a minute since I worked with this code, and got the original error. 

I was able to repro with your DRF tip, where i got this on `main` with the :

```
choices=[(x.name, x.value) for x in MyFakeModelWithChoiceField.ChoiceType]
```

```
FAILED graphene_django/rest_framework/tests/test_mutation.py::test_perform_mutate_success_with_enum_choice_field - assert [ErrorType(field='choiceType', messages=[ErrorDetail(string='"asdf" is not a valid choice.', code='invalid_choice')])] is None
```

Reply:
@sjdemartini apologies, its been a minute since I worked with this code, and got the original error. 

I was able to repro with your DRF tip, where i got this on `main` with the :

```
choices=[(x.name, x.value) for x in MyFakeModelWithChoiceField.ChoiceType]
```

- https://github.com/graphql-python/graphene-django/pull/1431/commits/b1494da6be8adedbe3fef9871a7ae958e845d40d

```
FAILED graphene_django/rest_framework/tests/test_mutation.py::test_perform_mutate_success_with_enum_choice_field - assert [ErrorType(field='choiceType', messages=[ErrorDetail(string='"asdf" is not a valid choice.', code='invalid_choice')])] is None
```